vcpkg/consume/manifest-mode.md

```diff
@@ -0,0 +1,295 @@
+  (omitted)
+SetModuleDependencies:
+  Creating directory "x64\Debug\manifest.ceffc6eb_MD.tlog\".
+VcpkgTripletSelection:
+  Using triplet "x64-windows" from "D:\projects\manifest-example\vcpkg_installed\x64-windows\x64-windows\"
+  Using normalized configuration "Debug"
+VcpkgInstallManifestDependencies:
+  Installing vcpkg dependencies to D:\projects\manifest-example\vcpkg_installed\x64-windows\
+  Creating directory "D:\projects\manifest-example\vcpkg_installed\x64-windows\".
+  "D:\vcpkg\vcpkg.exe" install  --x-wait-for-lock --triplet "x64-windows" --vcpkg-root "D:\vcpkg\" "--x-manifest-root=D:\projects\manifest-example\" "--x-install-root=D:\projects\manifest-example\vcpkg_installed\x64-windows\"
+  "D:\vcpkg\vcpkg.exe" install  --x-wait-for-lock --triplet "x64-windows" --vcpkg-root "D:\vcpkg\" "--x-manifest-root=D:\projects\manifest-example\" "--x-install-root=D:\projects\manifest-example\vcpkg_installed\x64-windows\"
+  Detecting compiler hash for triplet x64-windows...
+  The following packages will be built and installed:
+      cxxopts:x64-windows -> 3.1.1
+      fmt:x64-windows -> 10.0.0
+      range-v3:x64-windows -> 0.12.0#1
+    * vcpkg-cmake:x64-windows -> 2023-05-04
+    * vcpkg-cmake-config:x64-windows -> 2022-02-06#1
+  (omitted)
+ClCompile:
+  (omitted)
+Link:
+  (omitted)
+AppLocalFromInstalled:
+  pwsh.exe -ExecutionPolicy Bypass -noprofile -File "D:\vcpkg\scripts\buildsystems\msbuild\applocal.ps1" "D:\projects\manif
+  est-mode-msbuild\x64\Debug\manifest-example.exe" "D:\projects\manifest-example\vcpkg_installed\x64-windows\x64-windows\debug\bin"
+  "x64\Debug\manifest.ceffc6eb.tlog\manifest-example.write.1u.tlog" "x64\Debug\vcpkg.applocal.log"
+  D:\projects\manifest-example\x64\Debug\fmtd.dll
+FinalizeBuildStatus:
+  (omitted)
+Done Building Project "D:\projects\manifest-example\manifest-example.vcxproj" (default targets).
+
+Done Building Project "D:\projects\manifest-example\manifest-example.sln" (default targets).
+
+Build succeeded.
+```
+
+### [Visual Studio](#tab/visual-studio)
+
+> [!div class="nextstepaction"]
+> [Learn more about using vcpkg from Visual Studio](../users/buildsystems/msbuild-integration.md)
+
+### Step 2.1: Enable user-wide integration (Visual Studio)
+
+To use [vcpkg in your Visual Studio projects](../users/buildsystems/msbuild-integration.md) run the
+following command:
+
+```Console
+vcpkg integrate install
+```
+
+This integration method automatically adds vcpkg-installed packages to the project's Include Directories,
+Link Directories, and Link Libraries. Additionaly, this creates a post-build action that ensures
+that any required DLLs are copied in the build output folder. This works for all VS2015, VS2017,
+VS2019, and VS2022 projects.
+
+### Step 2.2: Enable vcpkg manifests
+
+By default, manifest mode is disabled in Visual Studio projects. To enable manifests, open the
+project's _Properties_ window and set _Use Vcpkg Manifest_ to _Yes_ in the _vcpkg_ tab.
```

Comment:
Do we know whether these italicized terms are translated in different languages in Visual Studio? If so, the code style may be more appropriate here so they don't get translated here.